contracts/booking-token/BookingToken.sol

```diff
@@ -407,12 +856,19 @@ contract BookingToken is
      * @notice Override transferFrom to check if token is reserved. It reverts if
      * the token is reserved.
      */
-    function transferFrom(address from, address to, uint256 tokenId) public override(ERC721Upgradeable, IERC721) {
+    function transferFrom(
+        address from,
+        address to,
+        uint256 tokenId
+    ) public virtual override(ERC721Upgradeable, IERC721) {
         // Verify that the token is transferable (i.e. not reserved)
         checkTransferable(tokenId);
         super.transferFrom(from, to, tokenId);
     }
 
+    // FIXME: Do we need to override the safeTransferFrom function? It's already
```

Comment:
Does this needs to be resolved before merge?

Reply:
Yes, I've tested the transfer functions and removed the `safeTransferFrom` override. It's not needed. Pushed the changes.